README.md

```diff
@@ -30,7 +30,11 @@ The following are instructions on how to build from source.
 ### Prerequisites
 
 Currently only supports Linux. Windows and MacOS support is planned.
-We need access to a full archive Ethereum node operating on the network matching the StarkNet network you wish to run. Currently this is either Goerli or Mainnet.
+We need access to a full Ethereum node operating on the network matching the StarkNet network you wish to run. Currently this is either Goerli or Mainnet.
+
+| :point_up:    | If using Infura as an L1 provider, you will need access to their archive node facilities. This is because `pathfinder` requires access to the full log history. |
```

Comment:
```suggestion
| :warning:    | If using Infura as an L1 provider, you will need access to their archive node facilities. This is because `pathfinder` requires access to the full log history. |
```

Or maybe :exclamation: ? My emoji-fu is not so strong.

Reply:
Lets ask the expert... @CHr15F0x?

Reply:
- :warning: points to the :arrow_right: current line :arrow_left:  while (*)

in the line above :thinking:
- (*) :point_up: seems to indicate something's interesting